tests/integ/modin/frame/test_quantile.py

```diff
@@ -131,9 +131,8 @@ def test_quantile_datetime_negative():
         snow_df.quantile(numeric_only=False)
 
 
-@pytest.mark.xfail(
+@pytest.mark.skip(
```

Comment:
It has to be skipped because without sql count check it doesn't fail. This xfail is actually a skip here because it will definitely fail due to `@sql_count_checker(query_count=0)`

Reply:
I see the point about why this test will fail, but I am curious is this causing any problem to the CTE tests? I thought you skip all sql_counter check already

Reply:
xfail with strict=True expects this test to fail because of query_count=0. When we skip sql counter check in with CTE enabled, this test will not fail. Then xfail will "fail" this test consistently.

Reply:
I see. Thanks for the explanation!